File: API/Features/Cashiers/Transactions/ViewModels/CashierListVM.cs
using API.Infrastructure.Classes;

namespace API.Features.Cashiers.Transactions {

    public class CashierListVM {

        public string Id { get; set; }
        public SimpleEntity Company { get; set; }
        public SimpleEntity Safe { get; set; }
        public string Date { get; set; }
        public string Entry { get; set; }
        public decimal Amount { get; set; }
        public decimal Debit { get; set; }
        public decimal Credit { get; set; }
        public string Remarks { get; set; }
        public bool HasDocument { get; set; }
        public string DocumentName { get; set; }
        public string PutAt { get; set; }

    }

}

File: FrontEnd/src/app/features/cashiers/transactions/classes/view-models/list/cashier-list-vm.ts
import { SimpleEntity } from 'src/app/shared/classes/simple-entity'

export interface CashierListVM {

    id: string
    date: string
    company: SimpleEntity
    safe: SimpleEntity
    remarks: string
    debit: string
    credit: string
    putAt: string
    hasDocument: boolean

}
